src/main/java/com/plugin/frontend/actions/FormatAreaAction.java

```diff
@@ -65,26 +71,42 @@ public void actionPerformed(@NotNull AnActionEvent e) {
 
         try {
             javaFormatter.formatPartsOfFile(new FileChanges(new File(file.getPath()), new ArrayList<>() {{
-                add(new Pair<>(finalStartLine, finalEndLine));
+                add(new Pair<>(finalStartLine, finalEndLine)); // объявить здесь переменные - не получилось
             }}));
         } catch (FormatterException | IOException exception) {
             exception.printStackTrace();
         }
     }
 
-    private int changeCount(int offset) throws IOException {
+    //position 0 - start; position 1 - end
+    private zxc getLineNumber(int offset, boolean position, LineNumberReader r, int count) throws IOException {
         int line = 0;
         int crutch = 0;
```

Comment:
This variable is useless here. U use var one time. Convert to bool

Reply:
Ask the help from the explanatory team